src/lib/api/archive.repo.ts

```diff
@@ -0,0 +1,28 @@
+import { PermanentApiRequestData, PermanentApiResponseData } from '../model';
+
+import { BaseRepo } from './base.repo';
+
+export type ArchiveResponse = PermanentApiResponseData<'ArchiveVO'>;
+
+export class ArchiveRepo extends BaseRepo {
+  public getByArchiveNbr(archiveNbr: string) {
+    const requestData: PermanentApiRequestData = {
+      ArchiveVO: {
+        archiveNbr,
+      },
+    };
+    return this.request<ArchiveResponse>('/archive/getByArchiveNbr', [
+      requestData,
+    ]);
+  }
+
+  public change(archiveNbr: string) {
```

Comment:
I would recommend this be a more explicit name -- e.g. `changeArchive` -- also it would be great to add a docblock to these methods.

Reply:
The methods on any `Repo` aren't meant to be exposed to the user, do we still want to document internally?

Reply:
FWIW I do think the more specific name would be helpful for internal code legibility, but I also appreciate that you're currently mapping the method names to the url pattern.

Fine leaving it as is, but I would suggest that overall pattern creates an unnecessary coupling between API design and SDK definition so might be worth moving away from linking the names overall!